packages/messaging/lib/index.js

```diff
@@ -483,6 +483,10 @@ class FirebaseMessagingModule extends FirebaseModule {
     // No-op as web platform isn't implemented
     return true;
   }
+
+  experimentalSetDeliveryMetricsExportedToBigQueryEnabled(enable) {
+    // No-op as web platform isn't implemented
```

Comment:
🤔  the idea of the modular API in react-native-firebase is not to no-op things that web doesn't implement or that we haven't implemented yet, rather it is to expose all available functionality in a way that is web-compatible.

In this case, we actually have an implementation available that works so I am not sure why the no-op here?

Should be similar to existing typing of 
https://github.com/invertase/react-native-firebase/blob/main/packages/messaging/lib/index.d.ts#L1014

with implementations callable like
https://github.com/invertase/react-native-firebase/blob/97a5523bac4570ba7b47fbfdbb22348f38246d11/packages/messaging/lib/index.js#L135

and 

https://github.com/invertase/react-native-firebase/blob/97a5523bac4570ba7b47fbfdbb22348f38246d11/packages/messaging/lib/index.js#L441

(that is, just delegating to the existing implementations which actually work, but using the firebase-js-sdk v9's `experimentalNNN` API symbol ? )